schema/connector-discovery-schema.json

```diff
@@ -0,0 +1,370 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "title": "Connector discovery schema",
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
Almost all places the schema doesn't allow authors to include unknown random properties.
I think it would help find issues in the discovery quickly.

Reply:
No, see chat discussion.